requirements.in

```diff
@@ -1,6 +1,6 @@
 connexion
 flask_restx
-funding-service-design-utils
+funding-service-design-utils==1.0.12
```

Comment:
I don't think you need to specify the utils version in this file. Would be worth checking with a Dev first.

Reply:
No worries, we can always revert it back if needed.